int64 dxl_spd1
int64 dxl_spd2
int64 dxl_spd3